cloudify_types/cloudify_types/component/operations.py

```diff
@@ -333,3 +333,19 @@ def refresh(**kwargs):
                      deployment.get('id') or
                      ctx.instance.id)
     populate_runtime_with_wf_results(client, deployment_id)
+
+
+@operation(resumable=True)
+@errors_nonrecoverable
+def check_drift(**kwargs):
+    # We might need an endpoint to get the drift and the status summary for
```

Comment:
getting the deployment already does this; and in cloudify-cosmo/cloudify-common#1070 I expose those values on the restclient object